.github/workflows/release-service-templates.yml

```
@@ -93,3 +94,10 @@ jobs:
           acrRegistry: ${{ env.ACR_REGISTRY }}
           acrUsername: ${{ env.ACR_USERNAME }}
           acrPassword: ${{ secrets.TEMPLATES_API_ACR_PASSWORD }}
+
+      - name: Trigger image tag update
+        uses: keboola/kbc-stacks/.github/actions/trigger-image-tag-update@main
+        with:
+          helm-chart: "templates-api"
+          image-tag: ${{ env.IMAGE_TAG }}
+          github-app-private-key: ${{ secrets.GITOPS_KBC_STACKS_TRIGGER_APP_PVK }}
```

Missing new line at the end of a file... Maybe you're missing .editorconfig plugin in your editor?

```suggestion
          github-app-private-key: ${{ secrets.GITOPS_KBC_STACKS_TRIGGER_APP_PVK }}

```